### PR TITLE
Initial implementation for ListMetadata

### PR DESCRIPTION
**Pull request by rafaeling**

### CODE REVIEW

**Comment by lukasmittag** [comment]
markdown linter fails

File: databroker/src/grpc/kuksa_val_v2/val.rs [hunk 1]
```diff
@@ -84,11 +86,88 @@ impl proto::val_server::Val for broker::DataBroker {
...
+                                data_type: proto::DataType::from(entry_metadata.data_type.clone())
+                                    as i32,
+                                entry_type: proto::EntryType::from(
+                                    entry_metadata.entry_type.clone(),
+                                ) as i32,
+                                description: Some(entry_metadata.description.clone()),
+                                comment: None,
+                                deprecation: None,
+                                unit: entry_metadata.unit.clone(),
+                                value_restriction: None,
```
Comment (lukasmittag) [inline databroker/src/grpc/kuksa_val_v2/val.rs:150]: why are some things None shouldn't we extend databroker to these attributes? e.g. add them to DataEntry?

File: databroker/src/grpc/kuksa_val_v2/val.rs [hunk 2]
```diff
@@ -460,4 +539,130 @@ mod tests {
...
+        match proto::val_server::Val::list_metadata(&broker, wildcard_req)
+            .await
+            .map(|res| res.into_inner())
+        {
+            Ok(list_response) => {
+                let entries_size = list_response.metadata.len();
+                assert_eq!(entries_size, 2);
+            }
+            Err(_status) => panic!("failed to execute get request"),
+        }
```
Comment (lukasmittag) [inline databroker/src/grpc/kuksa_val_v2/val.rs:593]: add another test with only one asteriks?

File: databroker/src/grpc/kuksa_val_v2/val.rs (hunk 1 above)
Comment (rafaeling) [reply databroker/src/grpc/kuksa_val_v2/val.rs:150]: Maybe we should bring this topic to API discussions. It seems to be also present in V1 but never used

File: databroker/src/grpc/kuksa_val_v2/val.rs (hunk 2 above)
Comment (rafaeling) [reply databroker/src/grpc/kuksa_val_v2/val.rs:601]: You are right, extended unit test to support one asteriks.

**Review by lukasmittag — APPROVED** [review]
LGTM

**Pull request merged**